main/res/values/strings.xml

```diff
@@ -1261,12 +1261,12 @@
     <string name="cache_advanced">Advanced</string>
     <string name="cache_waypoints">Waypoints</string>
     <plurals name="waypoints">
-        <item quantity="zero">%d waypoints</item>
-        <item quantity="one">%d waypoint</item>
-        <item quantity="two">%d waypoints</item>
-        <item quantity="few">%d waypoints</item>
-        <item quantity="many">%d waypoints</item>
-        <item quantity="other">%d waypoints</item>
+        <item quantity="zero">waypoints (%d)</item>
+        <item quantity="one">waypoint (%d)</item>
```

Comment:
IMHO it should always be "Waypoints" as the number inside the "()" is no longer directly related to the name in front.

Reply:
So no string array is necessary at all anymore, which also saves time for our translators ;)

Reply:
Would be ok for English and German, but I don't know about other languages, whether there is sort of a "generic plural" as well.

Reply:
Also for german it could formally correct still be "Wegpunkt (1)" but "Wegpunkte (2)"...

Reply:
I think it's not about a "generic plural" but the fact, that the page can show multiple waypoints and not only a single one.

And the number now is totally separated from the generic page name. 

For "logs" we also do not only display "log" if there's only one log for the cache.

Reply:
I understand that the page can display several waypoints, and because of that one could use a generic "waypoints". But I don't know if there is such a "generic plural" in all languages. "generic" in the sense of "not bound to a specific number > 1". For English and German there is no difference if it's 2, 5, 100 of 1 million waypoints, whereas in other language the translation for "waypoints" has to change in those cases.